jetty9/src/main/docker/root.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+<Configure id="webapp" class="org.eclipse.jetty.webapp.WebAppContext">
+  <Set name="contextPath">/root</Set>
+  <Set name="war"><Property name="jetty.webapps" default="."/>/root.war</Set>
+  <Set name="extractWAR">true</Set>
+  <Set name="tempDirectory"><Property name="jetty.base" default="."/>/webapps/root-exploded</Set>
```

Comment:
the standard for in situ extracting is to expand to webapps/root/ 
Having a temporary directory is a separate matter, although temporary extraction is done to the temporary directory.